torchaudio/__init__.py

```diff
@@ -34,6 +35,26 @@
     pass
 
 
+@_mod_utils.deprecated(
+    "Please remove the function call. Resource initialization is automatically handled")
+def initialize_sox() -> int:
+    """Initialize sox effects.
+
+    This function is deprecated. See ``torchaudio.sox_effects.init_sox_effects``
+    """
+    _init_sox_effects()
+
+
+@_mod_utils.deprecated(
+    "Please remove the function call. Resource clean up is automatically handled")
```

Comment:
nit: In case we would indeed move to deprecate this function: "Please remove the function call to ``torchaudio.shutdown_sox``. Resource clean up is now automatically handled. In the unlikely event that you need to manually shutdown sox, please use ``torchaudio.sox_effects.shutdown_sox_effects``."